packages/db/prisma/seed.ts

```diff
@@ -199,19 +202,9 @@ async function main() {
     blobsOnTxs.some((bot) => bot.blobHash === b.versionedHash),
   );
 
-  const uploadBlobsPromise = blobs.map(async (b) => {
-    const [blobExists] = await storage
-      .bucket(BUCKET_NAME)
-      .file(b.gsUri)
-      .exists();
-
-    if (!blobExists) {
-      await storage
-        .bucket(BUCKET_NAME)
-        .file(buildGoogleStorageUri(b.versionedHash))
-        .save(b.data);
-    }
-  });
+  const uploadBlobsPromise = uniqueBlobs.map(async (b) =>
```

Comment:
I was not using the `UniqueBlobs` here to make sure we only store on swarm the blobs that we are actually storing on the database. If I'm not mistaken might be cases were some `UniqueBlobs` will not be used. Anyways is an edge case, we can go ahead with the current script.